data/remnant/remnant events.txt

```diff
@@ -129,6 +157,34 @@ event "korath hunting"
 
 
 
+event "korath rescue: nenia small"
+	system Nenia
+		add fleet "Korath Rescue" 2000
```

Comment:
This is a fairly low period for a system so far from home.

Reply:
This isn't a random patrol, it's a response to a specific event in a known system.